Sources/FluentPostgresDriver/FluentPostgresConfiguration.swift

```diff
@@ -81,7 +81,9 @@ extension DatabaseConfigurationFactory {
         decodingContext: PostgresDecodingContext<some PostgresJSONDecoder>,
         sqlLogLevel: Logger.Level = .debug
     ) -> DatabaseConfigurationFactory {
-        .init {
+        let configuration = FakeSendable(wrappedValue: configuration)
```

Comment:
Why isn't configuration Sendable?

Reply:
Too much mutability. The correct solution is to do what was done in PostgresKit, deprecating the old one and making a completely new type, but that's more work than I'm willing to do this late in the lifetime of Fluent 4. This is more of a workaround for the lack of `transferring`; the configuration doesn't actually need to be `Sendable` conceptually.